grpc-spring-boot-starter/src/main/java/org/lognet/springboot/grpc/GRpcServerRunner.java

```diff
@@ -12,20 +15,18 @@
 import org.springframework.core.type.StandardMethodMetadata;
 
 import java.lang.annotation.Annotation;
-import java.util.Collection;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
+import java.util.*;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
 /**
- *  Hosts embedded gRPC server.
+ * Hosts embedded gRPC server.
  */
 @Slf4j
-public class GRpcServerRunner implements CommandLineRunner,DisposableBean  {
-
-
+public class GRpcServerRunner implements CommandLineRunner, DisposableBean {
+    private final Set<String> serviceList = new ConcurrentSet<>();
```

Comment:
> As per netty package, i think it's already included in the dependency.
Yes, it's included, but it's from `internal` package, please use standard concurrent set from JDK (`ConcurrentSkipListSet`  maybe ?)